app/services/record_declarations/retained.rb

```diff
@@ -8,5 +8,9 @@ module Retained
       validates :evidence_held, presence: { message: I18n.t(:missing_evidence_held) }
       validates :evidence_held, inclusion: { in: :valid_evidence_types, message: I18n.t(:invalid_evidence_type) }, allow_blank: true
     end
+
+    def valid_evidence_types
+      self.class.valid_evidence_types
+    end
```

Comment:
I really think this pattern makes things harder to understand than it needs to be. If the method is a class method, we should make it obvious where it is used.